src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -1,5 +1,7 @@
 package core.basesyntax;
 
+import java.util.Objects;
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/equals-and-hashcode/equals-and-hashcode#dont-use-methods-of-the-objectsclass